### PR TITLE
scheduler: fix error `nextRecover shouldn't be later than nextStart`

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Signed-off-by: xiang <xiang13225080@163.com>

### What problem does this PR solve?

fix issue https://github.com/chaos-mesh/chaos-mesh/issues/1207

it is easy to meet this error when create a chaos test with "scheduler": {"cron":"*/1 * * * * "}, "duration": "50s"

https://github.com/chaos-mesh/chaos-mesh/pull/1221 is done a similar thing, but it is hard to cherry pick to release-1.0

### What is changed and how does it work?

when the chaos is the first time run, set the NextStart for it

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by fewdan — APPROVED** [review]
LGTM

**Comment by cwen0** [comment]
@WangXiangUSTC PTAL

File: test/e2e/chaos/basic.go [hunk 1]
```diff
@@ -1509,6 +1509,8 @@ selector:
 						return false
 					}
 
+					time.Sleep(time.Second)
```
Comment (WangXiangUSTC) [inline test/e2e/chaos/basic.go:1512]: according to PR #1069 comment

**Pull request merged**